src/dune_rules/coq_rules.ml

```diff
@@ -241,7 +311,7 @@ let parse_coqdep ~dir ~(boot_type : Bootstrap.t) ~coq_module
       :: deps )
 
 let deps_of ~dir ~boot_type coq_module =
-  let stdout_to = Coq_module.obj_file ~obj_dir:dir coq_module Dep in
+  let stdout_to = Coq_module.dep_file ~obj_dir:dir coq_module in
```

Comment:
I've kept the dep file as its own function due to a planned optimization: we hope to call coqdep once for theory as this will provide a significant speed up, unfortunately coqdep needs both to scan the FS and have the sources in place so the finer granularity is not an advantate.

It would be great to have some support akin `-modules` in OCaml, but that's gonna take some non-trivial changes upstream.